tests/sklearn/test_sklearn_models.py

```diff
@@ -151,6 +151,28 @@ def get_n_bits_non_correctness(model_class):
     return n_bits
 
 
+def fit_and_compile(model, x, y):
+    """Fit the model and compile it."""
+
+    with warnings.catch_warnings():
+        # Sometimes, we miss convergence, which is not a problem for our test
+        warnings.simplefilter("ignore", category=ConvergenceWarning)
+        model.fit(x, y)
+
+    model.compile(x)
+
+
+def get_random_samples(x, n_sample):
+    """Selects `n_sample` random elements from a 2D NumPy array."""
```

Comment:
docstrings should be without s `Selects` -> `Select`, I'm surprised the conformance step didn't complain